### ISSUE TITLE
Need Info - How to square off position/s based on the net profit or net loss  

### ISSUE THREAD

**Issue by grsb007** [krishnavelu/alice_blue#40]
Wanted to build a system based and automatic square off positions based on net profit or net loss for example 

will be buying like following for example 
bn_call = alice.get_instrument_for_fno(symbol = 'BANKNIFTY', expiry_date=datetime.date(2019, 12, 19), is_fut=False, strike=30000, is_CE = True)
bn_put = alice.get_instrument_for_fno(symbol = 'BANKNIFTY', expiry_date=datetime.date(2019, 12, 19), is_fut=False, strike=30000, is_CE = False)

If in case of tool profit or loss is more than 1000 rs /X amount wanted to square off the positions. 
So what I wanted to know which API from AB can give this/this type of information it provides.
Could you, please provide some sample inputs on this.

**Comment by krishnavelu** [comment]
I cannot help anything on your own strategy. Restrict the questions only related to this library and its functionality.
Go though the complete documentation, especially [this](https://github.com/krishnavelu/alice_blue#create-aliceblue-object). This will give m2m.
```
print(alice.get_balance()) # get balance / margin limits
print(alice.get_profile()) # get profile
print(alice.get_daywise_positions()) # get daywise positions
print(alice.get_netwise_positions()) # get netwise positions
print(alice.get_holding_positions()) # get holding positions
```

**Comment by RajeshSivadasan** [comment]
Hi @krishnavelu , I was checking on the way to get MTM value by using alice.get_balance() but could not get while the order was open. I was expecting the values in the unrealised_m2m attribte. Once I closed my position I could see the value in realised_m2m attribute. Can you please guide on the proper way to get runtime MTM values as we see in ant web/mobile: I have provided the output of alice.get_balance() in both the scenarios:

This is during open position:
{'status': 'success', 'message': '', 'data': {'cash_positions': [{'utilized': {'var_margin': '0.00', 'unrealised_m2m': '-0.00', 'span_margin': '0.00', 'realised_m2m': '-0.00', 'premium_present': '0', 'pay_out': '0.00', 'multiplier': '1.00', 'exposure_margin': '0.00', 'elm': '0.00', 'debits': '7933.44'}, 'segment': 'ALL', 'net': '18137.04', 'category': 'ABFS-COMMON', 'available': {'pay_in': '0', 'notionalCash': '0.000000', 'direct_collateral_value': '0.00', 'credits': '26070.48', 'collateral_value': '0.00', 'cashmarginavailable': '26070.48', 'adhoc_margin': '0.000000'}}, {'utilized': {'var_margin': '0.00', **'unrealised_m2m': '-0.00'**, 'span_margin': '0.00', **'realised_m2m': '-0.00'**, 'premium_present': '0', 'pay_out': '0.00', 'multiplier': '1.00', 'exposure_margin': '0.00', 'elm': '0.00', 'debits': '0.00'}, 'segment': 'COM', 'net': '0.00', 'category': 'NO_VAL', 'available': {'pay_in': '0', 'notionalCash': '0.000000', 'direct_collateral_value': '0.00', 'credits': '0.00', 'collateral_value': '0.00', 'cashmarginavailable': '0.00', 'adhoc_margin': '0.000000'}}]}}

This is after all the positions were closed:
{'cash_positions': [{'utilized': {'var_margin': '0.00', 'unrealised_m2m': '-0.00', 'span_margin': '0.00', 'realised_m2m': '-0.00', 'premium_present': '0', 'pay_out': '0.00', 'multiplier': '1.00', 'exposure_margin': '0.00', 'elm': '0.00', 'debits': '0.00'}, 'segment': 'COM', 'net': '0.00', 'category': 'NO_VAL', 'available': {'pay_in': '0', 'notionalCash': '0.000000', 'direct_collateral_value': '0.00', 'credits': '0.00', 'collateral_value': '0.00', 'cashmarginavailable': '0.00', 'adhoc_margin': '0.000000'}}, {'utilized': {'var_margin': '0.00', **'unrealised_m2m': '-0.00'**, 'span_margin': '0.00', **'realised_m2m': '4676.25'**, 'premium_present': '0', 'pay_out': '0.00', 'multiplier': '1.00', 'exposure_margin': '0.00', 'elm': '0.00', 'debits': '-3801.16'}, 'segment': 'ALL', 'net': '31862.20', 'category': 'ABFS-COMMON', 'available': {'pay_in': '2000.000000', 'notionalCash': '0.000000', 'direct_collateral_value': '0.00', 'credits': '28061.04', 'collateral_value': '0.00', 'cashmarginavailable': '26061.04', 'adhoc_margin': '0.000000'}}]}

Thanks!

**Comment by krishnavelu** [comment]
Try daywise/netwise positions
